.gitignore

```diff
@@ -0,0 +1,3 @@
+/node_modules
+.hintrc
+.stylelint.json
```

Comment:
Remove these `.hintrc` and `.stylelintrc,json` files from gitiignore before you merge

Reply:
Done! Thanks for your time.

Reply:
Thank you.